src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IBinaryOperatorExpression.cs

```diff
@@ -898,5 +903,1194 @@ void M(bool a, bool b, bool c, bool d, bool e)
 
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void LogicalFlow_06()
```

Comment:
>LogicalFlow [](start = 20, length = 11)

We are actually checking the shape of the graph for non-logical binary operators.

Reply:
Fixed naming.

---
In reply to: [164228063](https://github.com/dotnet/roslyn/pull/24465#discussion_r164228063) [](ancestors = 164228063)